src/gmm_utils.erl

```diff
@@ -124,6 +124,8 @@ owner_of(Vertex) ->
 
 %% Edges
 
+%% Other functions
```

Comment:
Tego komentarza nie rozumiem. Już jest komentarz że to o krawędziach (bo tak jest).